app/v1/controllers/AmendPensionChargesController.scala

```diff
@@ -64,16 +64,20 @@ class AmendPensionChargesController @Inject()(val authService: EnrolmentsAuthSer
           logger.info(s"[${endpointLogContext.controllerName}][${endpointLogContext.endpointName}] - " +
             s"Success response received with CorrelationId: ${responseWrapper.correlationId}")
 
+          val amendResponse = amendPensionsHateoasBody(appConfig, nino, taxYear)
```

Comment:
val name might be better as something like responseWrapperWithHateos as it describes what it contains and why you are not just using the responseWrapper from the service.